configure.ac

```diff
@@ -402,23 +402,15 @@ AS_IF([test "x${enable_checks}" != "xno" && {
        test "x$OCAML_PKG_swhid_core" = "xno" ||
        test "x$CPPO" = "x" ||
        test "x$OCAML_PKG_mccs$MCCS_ENABLED" = "xnotrue";}],[
-  AS_IF([test "x${with_vendored_deps}" = "xno"],[
-    AC_MSG_ERROR([Dependencies missing and --without-vendored-deps given; override with --disable-checks])
-  ],[
-    AS_IF([test "x${with_vendored_deps}" != "xyes"],[
-      echo "============================================================================"
-      echo "Some dependencies are missing. Vendored mode has been automatically enabled."
-      echo "The sources for opam's dependencies will be extracted in src_ext and built."
-      echo "============================================================================"
-    ])
-    AC_SUBST(VENDORED,"true")
+  AS_IF([test "x${with_vendored_deps}" != "xyes"],[
+    AC_MSG_ERROR([Dependencies missing. Use --with-vendored-deps or --disable-checks])
```

Comment:
I'd highlight more this part, as previous opposite message, surrounded with `====[...]`

Reply:
I don't think this is needed. The previous message were surrounded by `===` because it was a simple notice and does not end the configure script.

`AC_MSG_ERROR` makes it very clear it is an error and makes the configure script fail. The output will look like this:
```
$ ./configure
[...]
checking whether g++ really is a C++ compiler... no

configure: error: Dependencies missing. Use --with-vendored-deps or --disable-checks.
```